### DISCUSSION TITLE
[@types/react-native] Missing announceForAccessibilityWithOptions inside AccessibilityInfo

### DISCUSSION THREAD

**Discussion by megancallenbach** [DefinitelyTyped/DefinitelyTyped#64631] [Issues with a @types package]
I'm getting typescript issues because the @types/react native package doesn't have the announceForAccessibilityWithOptions function. Introduced in version 0.68 of React Native.

https://reactnative.dev/docs/accessibilityinfo#announceforaccessibilitywithoptions

[a comment by an automated account is not shown]

**Comment by eps1lon** [comment]
Thanks for reporting! This should be fixed if you're on `react-native@0.71` and us their built-in types (i.e. don't have `@types/react-native`) installed.
For versions before 0.71 that still need `@types/react-native` I've filed https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64634
